airflow/providers/ssh/hooks/ssh.py

```diff
@@ -282,13 +282,14 @@ def get_conn(self) -> paramiko.SSHClient:
             else:
                 pass  # will fallback to system host keys if none explicitly specified in conn extra
 
+        host_proxy = self.host_proxy_cmd and paramiko.ProxyCommand(self.host_proxy_cmd)
```

Comment:
This will return boolean, you could instead have.

```
@property
def host_proxy(self):
    return paramiko.ProxyCommand(self.host_proxy_cmd) if self.host_proxy_cmd else None
```

Reply:
Thanks.  I was thinking the same (making `host_proxy` a property).